### ISSUE TITLE
On macOS, the hotkeys to preview (ctrl-→ and ctrl-←) are already occupied by the system

### ISSUE THREAD

**Issue by GabrielBrodersen** [Canop/broot#497]
When using broot, the ctrl + right/left arrows appears to be important hotkeys to preview files and move between panes. However, on MacOS these hotkeys are already occupied by default to move between spaces (see System Preferences > Keyboard > Mission Control > Shotcuts, then look under ⌄ Mission Control ). 

Is there a way to remap them in broot?

**Comment by Canop** [comment]
Yes, you can rebind them, see https://dystroy.org/broot/conf_verbs/


But you might also be interested in this issue: https://github.com/Canop/broot/issues/480

**Comment by yesenadam** [comment]
Thank you @GabrielBrodersen for mentioning this! Googling the problem landed me here. I just turned off the 2 mission control shortcuts and panels now work.

**Comment by Canop** [comment]
Since broot 1.60, alt-arrows are bound to the same internals as ctrl-arrows, and on macOS the hints and help show the alt combinations, so nothing needs to be changed in the system settings anymore. See https://dystroy.org/broot/common-problems/#ctrl-arrows-or-alt-arrows. Closing.
